lib/RGBot.js

```diff
@@ -1214,7 +1328,7 @@ class RGBot extends EventEmitter {
         if (bestHarvestTool) {
             try {
                 await this.#bot.equip(bestHarvestTool, 'hand');
-                return this.#bot.heldItem;
+                return new Promise( resolve => this.#bot.heldItem);
```

Comment:
Wouldn't this never resolve? Should it be this? (not actually sure myself)
```typescript
return new Promise( resolve => resolve(this.#bot.heldItem));
```

Reply:
yep, my mistake.. fixing